multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/executors/TimeoutAsyncTaskExecutor.java

```diff
@@ -0,0 +1,32 @@
+package org.cloudfoundry.multiapps.controller.process.executors;
+
+import org.flowable.common.engine.impl.async.DefaultAsyncTaskExecutor;
+
+import java.util.concurrent.Future;
+
+public class TimeoutAsyncTaskExecutor extends DefaultAsyncTaskExecutor {
+    private final CancelExpiredTasksThread cancelExpiredTasksThread;
+
+    public TimeoutAsyncTaskExecutor(long timeoutInMillis, long waitTimeInMillis) {
+        executorNeedsShutdown = true;
```

Comment:
I think this is not needed, because it is set in DefaultAsyncTaskExecutor.start()